database/alembic/versions/7c7900da4510_v5_upgrade.py

```diff
@@ -0,0 +1,224 @@
+
+
+# revision identifiers, used by Alembic.
+revision: str = '7c7900da4510'
+down_revision: Union[str, None] = 'a1a7539b1a57'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSON
+
+def upgrade():
+    # Create tables in "extract" schema
+    op.create_table(
+        'assessment',
+        sa.Column('id', sa.String(), nullable=False),
+        sa.Column('pid', sa.String(), nullable=True),
+        sa.Column('creation_date', sa.TIMESTAMP(), server_default=sa.text('now()'), nullable=False),
+        sa.Column('update_date', sa.TIMESTAMP(), nullable=True),
+        sa.Column('assessmentstart', sa.TIMESTAMP(), nullable=True),
+        sa.Column('assessmentend', sa.TIMESTAMP(), nullable=True),
+        sa.Column('assessmenttypecode', sa.String(length=100), nullable=True),
+        sa.Column('assessmenttypecodestd', sa.String(length=100), nullable=True),
+        sa.Column('assessmenttypecodedesc', sa.String(length=100), nullable=True),
+        sa.Column('assessmentoutcomecode', sa.String(length=100), nullable=True),
+        sa.Column('assessmentoutcomecodestd', sa.String(length=100), nullable=True),
+        sa.Column('assessmentoutcomecodedesc', sa.String(length=100), nullable=True),
+        sa.PrimaryKeyConstraint('id'),
+        schema='extract'
+    )
+
+    op.alter_column('patientrecord', 'channelid', new_column_name='latestfilehash', schema='extract')
+
+    # Add columns to existing tables
+    op.add_column('causeofdeath', sa.Column('verificationstatus', sa.String(length=100)), schema='extract')
+    op.add_column('diagnosis', sa.Column('verificationstatus', sa.String(length=100)), schema='extract')
+    op.add_column('renaldiagnosis', sa.Column('biopsyperformed', sa.String(length=100)), schema='extract')
+    op.add_column('renaldiagnosis', sa.Column('verificationstatus', sa.String(length=100)), schema='extract')
+
+    # Create additional tables
+    op.create_table(
+        'dialysisprescription',
+        sa.Column('id', sa.String(length=20), nullable=False),
+        sa.Column('pid', sa.String(length=20), nullable=True),
+        sa.Column('creation_date', sa.TIMESTAMP(), server_default=sa.text('now()'), nullable=False),
+        sa.Column('update_date', sa.TIMESTAMP(), nullable=True),
+        sa.Column('enteredon', sa.TIMESTAMP(), nullable=True),
+        sa.Column('fromtime', sa.TIMESTAMP(), nullable=True),
+        sa.Column('totime', sa.TIMESTAMP(), nullable=True),
+        sa.Column('sessiontype', sa.String(length=5), nullable=True),
+        sa.Column('sessionsperweek', sa.Integer(), nullable=True),
+        sa.Column('timedialysed', sa.Integer(), nullable=True),
+        sa.Column('vascularaccess', sa.String(length=5), nullable=True),
+        sa.PrimaryKeyConstraint('id'),
+        schema='extract'
+    )
+
+    op.create_table(
+        'vascularaccess',
```

Comment:
I think this should already exist?

https://github.com/renalreg/ukrdc_resources_database/tree/main/schema/UKRDC/DDL

Reply:
Ignore the alembic stuff, I left it in in case it is something Andy wants but I didn't find it helpful.

Reply:
Can it all go in a separate branch then so it's less confusing?

Same with anything else which isn't meant to be used.